src/lib/config.js

```diff
@@ -30,7 +30,7 @@ class Config {
     this.timezone = 0;
 
     this.locale = 'enUS';
-    this.os = 'Mac';
+    this.os = 'Win';
```

Comment:
I see the following code, but I'm not sure I'm following why Warden fails if the identifier isn't one of 'Win' or 'OSX' -- to me, it looks like it just skips init'ing Warden altogether.

```
void WorldSession::InitWarden(BigNumber* k, std::string const& os)
{
    if (os == "Win")
    {
        _warden = new WardenWin();
        _warden->Init(this, k);
    }
    else if (os == "OSX")
    {
        // Disabled as it is causing the client to crash
        // _warden = new WardenMac();
        // _warden->Init(this, k);
    }
}
```